lib/cn/kvset_split.c

```diff
@@ -341,50 +341,48 @@ kblocks_split(
 
         /* split kblock at split_idx */
         err = kblock_split(&kbd, split_kobj, kblks, hlogs, vused);
-        if (err)
-            return err;
-        assert(kblks[LEFT].n_blks > 0 && kblks[RIGHT].n_blks > 0);
+
+        assert(err || (kblks[LEFT].n_blks > 0 && kblks[RIGHT].n_blks > 0));
```

Comment:
nit: swap the operands so that the most common case comes first.

Reply:
Yes. Done.